release/models/interfaces/openconfig-if-common.yang

```diff
@@ -0,0 +1,301 @@
+submodule openconfig-if-common {
+
+  yang-version "1";
+
+  belongs-to openconfig-interfaces {
+    prefix "oc-if";
+  }
+
+  import openconfig-yang-types { prefix oc-yang; }
+  import openconfig-extensions { prefix oc-ext; }
+  import openconfig-types { prefix oc-types; }
+
+  // meta
+  organization
+    "OpenConfig working group";
+
+  contact
+    "OpenConfig working group
+    netopenconfig@googlegroups.com";
+
+  description
+    "This sub-module contains groupings that are common across multiple
+    modules within interfaces folder. For example, these groups may be
+    used for interfaces and subinterfaces.";
```

Comment:
Note, that already `openconfig-if-ip` has to import `openconfig-interfaces`, and lots of models import `openconfig-interfaces` to use things like `base-interface-ref`.

Reply:
removed submodule and moved grouping into openconfig-interfaces.yang